docs/source/main_classes/trainer.rst

```diff
@@ -267,16 +267,16 @@ To deploy this feature:
    and make sure you have added the distributed launcher ``-m torch.distributed.launch
    --nproc_per_node=NUMBER_OF_GPUS_YOU_HAVE`` if you haven't been using it already.
 
-For example here is how you could use it for ``run_seq2seq.py`` with 2 GPUs:
+For example here is how you could use it for ``run_translation.py`` with 2 GPUs:
 
 .. code-block:: bash
 
-    python -m torch.distributed.launch --nproc_per_node=2 examples/seq2seq/run_seq2seq.py \
+    python -m torch.distributed.launch --nproc_per_node=2 examples/seq2seq/run_translation.py \
     --model_name_or_path t5-small --per_device_train_batch_size 1   \
     --output_dir output_dir --overwrite_output_dir \
     --do_train --max_train_samples 500 --num_train_epochs 1 \
     --dataset_name wmt16 --dataset_config "ro-en" \
-    --task translation_en_to_ro --source_prefix "translate English to Romanian: " \
+    --source_language en --target_language ro --source_prefix "translate English to Romanian: " \
```

Comment:
Wondering if we should always use language codes that work for mBART (so en_XX and ro_RO) so the same command works well for those checkpoints.

Reply:
`en_XX` appeared quite mysterious so I favoured the simpler ISO codes since it appears that's what we do currently anyway: `.split("_")[0]`

Reply:
That might not work. `MBartTokenizer` needs such custom language codes, and they `src_lang` and `tgt_lang` are set before doing `.split("_")[0]`

Reply:
yes I've updated the doc as per [this comment](https://github.com/huggingface/transformers/pull/10611#discussion_r590554222), but I'd welcome comment w.r.t this: 
https://github.com/huggingface/transformers/blob/7df219b11607662c4ca9ba65988a296dff550d28/examples/seq2seq/README.md#L183

I have a feeling you might know where to find this doc?

Reply:
The language codes can be found in the model card
https://huggingface.co/facebook/mbart-large-cc25